zebra/zebra_pbr.h

```diff
@@ -38,6 +38,8 @@ struct zebra_pbr_rule {
 	struct pbr_rule rule;
 
 	struct interface *ifp;
+
+	ns_id_t ns_id;
```

Comment:
you should also add ns_id_t ns_id to struct zebra_pbr_ipset too.

Reply:
I'll add the vrf_id to the hash